dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContentletLoader.java

```diff
@@ -561,11 +561,11 @@ public InputStream writeObject(final VelocityResourceKey key)
         long language = new Long(key.language);
         ContentletVersionInfo info = APILocator.getVersionableAPI().getContentletVersionInfo(key.id1, language);
 
-        if (!isLiveVersionAvailable(key, info) && language != defaultLang && APILocator.getLanguageAPI().canDefaultContentToDefaultLanguage()) {
+        if (isLiveVersionNotAvailable(key, info) && language != defaultLang && APILocator.getLanguageAPI().canDefaultContentToDefaultLanguage()) {
```

Comment:
maybe we should move this condition to a private method, this way the method name could give us more information about what it is doing here